src/main/java/core/basesyntax/service/OperationSelector.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.service;
+
+public interface OperationSelector {
+    void valueOperation(String key, int value);
```

Comment:
Think about names. Is it clear to understand what this method does? OperationSelector -> OperationHandler